nodes/src/nodes/llm_openai_api/IGlobal.py

```
@@ -0,0 +1,94 @@
+from rocketlib import IGlobalBase, OPEN_MODE, warning, debug
+from ai.common.config import Config
+from ai.common.chat import ChatBase
+import os
+import re
+
+
+class IGlobal(IGlobalBase):
+    chat: ChatBase | None = None
+
+    def validateConfig(self):
```

Ruff (0.15.7)

[warning] Too many branches (14 > 12)

(PLR0912)